### DISCUSSION TITLE
Telemundo Phoenix AZ (KTAZ-DT1 39.1)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#347] [Show and tell]
### Channel Name

Telemundo Arizona (KTAZ-DT1)

### Channel ID

KTAZDT1.us

### Stream URL

https://16live00.akamaized.net/TELEMUNDO/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Central302** [comment]
Access Denied.

**Comment by R4IN2007** [comment]
Its Works
![315_Chrome](https://user-images.githubusercontent.com/83226646/159784936-33fd7f4b-bd2c-434f-b206-1d66fe4416fb.jpg)

**Comment by Sphinxroot** [comment]
more here  seem to be a tvtap clone
https://github.com/MaxxGunner2022/iptvforu/blob/5ade4e05b499e7cc1eec37e0012e9ebd8cc16d52/iptv4u

they work but geo  

in kodi add |X-Forwarded-For=111.65.47.180 to link in playlistloader

**Comment by freearhey** [comment]
https://16live00.akamaized.net/TELEMUNDO/index.m3u8 (offline)
<img width="414" alt="image" src="https://user-images.githubusercontent.com/7253922/232226679-fc8e29c6-e7f9-497c-ad18-fd0889edf7c9.png">
